src/components/attendee/RetunableMediaElement/hooks.tsx

```diff
@@ -0,0 +1,51 @@
+import { useMemo } from "react";
+
+import { COLLECTION_RETUNABLE_MEDIA_ELEMENTS } from "settings";
+
+import { useRefiDocument } from "hooks/fire/useRefiDocument";
+
+import {
+  NotTunedSettings,
+  RetunableMediaElementSettings,
+  RetunableMediaSource,
+} from "./types";
+
+interface useRetunableMediaElementDataOptions {
```

Comment:
```suggestion
interface UseRetunableMediaElementDataOptions {
```

But I would go for a more holistic approach of `UseRetunableMediaElement ` type for the hook that has these options and returns ` {} & LoadStatus `

Reply:
Resolved in 40c9750b0